servatrice/src/servatrice.cpp

```diff
@@ -160,6 +160,14 @@ bool Servatrice::initServer()
         authenticationMethod = AuthenticationNone;
     }
 
+	bool maxUserLimitEnabled = settingsCache->value("security/enable_max_user_limit", false).toBool();
+	qDebug() << "Maximum user limit enabled: " << maxUserLimitEnabled;
+
+	if (maxUserLimitEnabled){
+		int maxUserLimit = settingsCache->value("security/max_users_total", 500).toInt();
+		qDebug() << "Maximum user limit: " << maxUserLimit;
```

Comment:
This is used to dump the information out to the console on server startup to inform the operator of the settings.  I simply followed the same code syntax involved as the other bits of information that is used to output to the console on startup.
Short answer, no it is supposed too be there.